Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/Microsoft.MixedReality.Toolkit.Providers.WindowsVoiceInput.asmdef

```diff
@@ -12,7 +12,7 @@
         "WindowsStandalone64"
     ],
     "excludePlatforms": [],
-    "allowUnsafeCode": false,
+    "allowUnsafeCode": true,
```

Comment:
Was this change to allow unsafe code intentional (and if so, what was the reason?) The rest of our asmdefs today don't have this, so I'm curious what motivated this change.

Reply:
accidental. should not have been changed

Reply:
fixed